### ISSUE TITLE
Black bar when using a huge profile picture

### ISSUE THREAD

**Issue by CiriousJoker** [Litarvan/lightdm-webkit-theme-litarvan#46]
I have a huge dark profile picture and when I press space (not before) and everything blurs, there's a blurred vertical.

The vertical line is placed in a way that it could be the edge of the square profile picture, that's resized to fit the window container.

Example screen:
```
+-------------------------------------+
|                 O          |        |
|               -----        |        |
|                ---         |        |
|                            |        |
|                            |        |
|                            |        |
|                            |        |
|                            |        |
+-------------------------------------+
```

**Comment by CiriousJoker** [comment]
I noticed that this bug is not present in the web version. There, no black bar appears, even when exchanging the loaded avatar for mine from this url:

https://i.imgur.com/MLZFRUO.jpg

Also, in the web version, there are shadows around the whole page probably because of the blur effect.

**Comment by Litarvan** [comment]
I'll try to debug this, lightdm-webkit2-greeter use a very old version of Webkit working very bad. And the new version (web-greeter) is definetly not stable (not even managed to build it)

**Comment by atm0915** [comment]
I have the same issue and my profile picture is only 96x96px

**Comment by CiriousJoker** [comment]
The issue still persists in the current 3.0.0-pre3.
Could this bed due to my screen resolution (2256x1504)?
Live web version still works no matter how far I zoom out.

**Comment by Litarvan** [comment]
Closed as duplicate of #73
